heartbeat/ocf-shellfuncs.in

```diff
@@ -989,6 +989,44 @@ ocf_stop_trace() {
 	set +x
 }
 
+# Helper functions to map from nodename/bundle-name and physical hostname
+# list_index_for_word "node0 node1 node2 node3 node4 node5" node4 --> 5
+# list_word_at_index "NA host1 host2 host3 host4 host5" 3      --> host2
+
+# list_index_for_word "node1 node2 node3 node4 node5" node7 --> ""
+# list_word_at_index "host1 host2 host3 host4 host5" 8      --> ""
+
+# attribute_target node1                                    --> host1
+function list_index_for_word() {
+	echo $1 | tr ' ' '\n' | awk -v x="$2" '$0~x {print NR}'
+}
+
+function list_word_at_index() {
+	echo $1 | tr ' ' '\n' | awk -v n="$2" 'n == NR'
+}
+
+function attribute_target() {
```

Comment:
Could you rename this `ocf_attribute_target`? The name is generic enough that it might be good to avoid collisions if possible. Also it clarifies where the function might be defined for someone reading the code.

Reply:
Make sense, will amend. Thanks!